Application/EdFi.Ods.WebService.Tests/Controllers/OpenApiMetadataControllerTests.cs

```diff
@@ -27,8 +27,11 @@ public async Task MetadataEndpointGetShouldBeValid()
             var json = await response.Content.ReadAsStringAsync();
 
             json.ShouldNotBeNullOrWhiteSpace();
-            Approvals.Verify(json);
```

Comment:
This seems like it would be a stable approval test, any reason to move this back to just checking for a non-empty array?

Reply:
i was seeing issues between tests that the results would be different. so i went to deserializing it to the model and checking